app/assets/stylesheets/components/_outcome-card.scss

```diff
@@ -96,8 +96,17 @@ $outcome-card-height: 185px;
 }
 
 .outcome-card-description {
-  @include margin($smallest-spacing null $small-spacing);
+  @include margin(0 null $small-spacing);
+  color: $medium-gray;
   font-family: $sans-serif-narrow;
-  font-size: $small-font-size;
   line-height: 1.4;
+  max-width: 35em;
```

Comment:
Are you already putting a threshold on the width by giving it's container the `$sensible-line-length`?